Update-docs (#155)
stellar/stellar-docs: docs/anchoring-assets/anchor-platform/sep24/faq.mdx

:::

> ### How to identify user account?

Reviewer: ```suggestion
### How to identify the user account?
```

> You should use `sub` field of the JWT token. For custodial wallet, this value will be in the format `account:memo`. Use memo to identity the user. For noncustodial wallet, simply use `sub` value itself, which will be equal to the user account.

Reviewer: ```suggestion
You should use the `sub` field of the JWT token. For custodial wallets, this value will be in the format `account:memo`. Use the memo to identity the user. For noncustodial wallets, simply use the `sub` value itself, which will be equal to the user account.
```

### How to identify wallet?

> You should use combination of `sub` and `data.client_domain` fields of the JWT token. For custodial wallet, `sub` value will be in the format `account:memo`. Use `account` id to identity the wallet. For noncustodial wallet, use the `data.client_domain` field. Note, that wallet must provide the `client_domain` beforehand as a part of SEP-10 authentication.

Reviewer: ```suggestion
You should use a combination of `sub` and `data.client_domain` fields of the JWT token. For custodial wallets, the `sub` value will be in the format `account:memo`. Use the account to identify the wallet. For noncustodial wallets, use the `data.client_domain` field. Note that the wallet must provide the `client_domain` beforehand as a part of SEP-10 authentication.
```
